src/client/app/strings/index.ts

```diff
@@ -50,7 +50,17 @@ export function Strings(): ClientStrings {
 }
 
 export function getDisplayLanguage() {
-    return (getRawDisplayLanguage() || 'en').toLowerCase();
+    let rawDisplayLanguage = getRawDisplayLanguage();
+    if (rawDisplayLanguage) {
+        rawDisplayLanguage = rawDisplayLanguage.toLowerCase();
+        for (let language of availableLanguages) {
+            if (rawDisplayLanguage === language.value) {
+                return rawDisplayLanguage;
+            }
+        }
+    }
+
+    return (rawDisplayLanguage || 'en').substr(0, 2);
```

Comment:
But I thought that in the pull request to add Chinese, we now have some places where we do use the region/locale.  Is that not so?  It just feels like we're trimming out possibly-useful information, not for English itself (since we only have one at the moment), but for another language that we might get from the `getRawDisplayLanguage()` method.